.github/workflows/packages.yml

```diff
@@ -28,3 +28,32 @@ jobs:
           path: |
             dist/*.tar.gz
             dist/*-manylinux1_*.whl
+
+  build-windows:
+    name: Build Windows packages
+    runs-on: windows-latest
+    strategy:
+      max-parallel: 100
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - name: Checkout repos
+      uses: actions/checkout@v1
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Build x86_64 packages
+      shell: cmd
+      run: |
+        pip install wheel || exit /b
+        python setup.py bdist_wheel
+
+    - name: Upload artifacts
+      uses: actions/upload-artifact@v1
+      with:
+        name: pymi_wheel_py${{ matrix.python-version }}
```

Comment:
Is this pattern correct?

Reply:
I can see the results here: https://github.com/dvarrazzo/py-setproctitle/runs/1500121685?check_suite_focus=true

Am I missing something?

Reply:
Ah ok. They didn't appear for a few minutes :)

Reply:
> Is this pattern correct?

My bad, I'll update the template.